README.md

```diff
@@ -12,3 +12,15 @@
 
 This repo contains environment definitions and scenarios for using DAML within
 the [Katacoda](https://www.katacoda.com) learning environment.
+
+# Ensuring that DAML projects used in the scenarios are up to date with the latest DAML SDK version
+
+When using external DAML projects in the scenario (e.g., via a downloaded `.tar` file) make sure that `daml.yaml` file references the DAML SDK version used in the Katacoda environment. You can do that by either
+
+- Creating a new project with `daml new` command and moving the project files into the folder
+- By changing the DAML SDK version in the `daml.yaml` file via shell and awk
```

Comment:
I'd prefer using `new`, because the awk/sed trick is fairly fragile.

Reply:
I've been using it, seems to work. I know you prefer `daml new`, that's why I have it as the first option